docs/userGuide/cliCommands.md

```diff
@@ -167,6 +167,48 @@ The caveat is that not building all pages during the initial process, or not reb
 
 <hr><!-- ========================================================================== -->
 
+### `archive` Command
+<br>
+
+**Format:** `markbind archive [options] [versionName] [archivePath]`
+
+**Alias:** `markbind ar`
+
+**Description:** Archives a version of the current site, which will not be affected by future changes to the site
+
+**Arguments:**
+
+<div id="archiveWarning">
+<box type="warning">
+
+Warning: If the folder at `<archivePath>/<versionName>` already exists, the contents will be overwritten and your previous files may be lost. Only do so if you need to replace all the archived files with the current site files.
+</box>
+</div>
+
+* `[versionName]`<br>
+  The name of the version, and the folder which the generated HTML files and assets will be stored in.<br>
+  {{ icon_example }} `version_1`
```

Comment:
Perhaps can mention that version numbers can follow [semver](https://semver.org/), and give a more realistic version example, e.g. `v.1.1.1` or `v1`

Reply:
I think whether or not users want to follow semantic versioning is up to them (this might not be so useful for a course website, for example) but I'll add the additional examples and change the example here like you suggest!